pkg/services/sqlstore/org_test.go

```diff
@@ -35,6 +35,44 @@ func TestAccountDataAccess(t *testing.T) {
 			So(len(query.Result), ShouldEqual, 3)
 		})
 
+		Convey("Given we have organizations, we can limit and paginate search", func() {
+			var err error
+			var cmd *models.CreateOrgCommand
+			ids := []int64{}
+
+			for i := 1; i < 4; i++ {
+				cmd = &models.CreateOrgCommand{Name: fmt.Sprint("Org #", i)}
+				err = CreateOrg(cmd)
+				So(err, ShouldBeNil)
+
+				ids = append(ids, cmd.Result.Id)
+			}
+
+			Convey("Should be able to limit search", func() {
+				query := &models.SearchOrgsQuery{Limit: 1}
+				err = SearchOrgs(query)
+
+				So(err, ShouldBeNil)
+				So(len(query.Result), ShouldEqual, 1)
+			})
+
+			Convey("Should be able to paginate search", func() {
+				query := &models.SearchOrgsQuery{Page: 0}
```

Comment:
Is there any point to this test case, since it's just using the default page and limit parameters?

Reply:
Thanks for the review! With this test, I was looking to test the defaults (and really should've dropped the `Page: 0` part). Should I remove it all together? 

Or should I make >1000 orgs to test that the default is still 1000 (just to make sure that previous behavior is still maintained)? That test seems a little extreme, but it does test for what I want to verify.